packages/discovery-provider/plugins/pedalboard/apps/solana-relay/src/routes/listen/listen.ts

```diff
@@ -191,20 +199,25 @@ export const listen = async (req: Request, res: Response) => {
         { userId, trackId, ip, timestamp, signature },
         'unauthorized request'
       )
-      return res.status(401).json({ message: 'Unauthorized Error' })
+      res.status(401).json({ message: 'Unauthorized Error' })
```

Comment:
FWIW I have `UnauthorizedError` `BadRequestError` etc that can be thrown and caught and passed through to the error handler middleware if you want and have the error handler do the response and logging and everything.

Slightly nicer because that shows as an error rather than successful request